Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -682,13 +682,54 @@ namespace O3DE::ProjectManager
 
         if (foundButton != m_projectButtons.end())
         {
-            (*foundButton).second->SetState(ProjectButtonState::Downloading);
+            (*foundButton).second->SetState(queueBuild ? ProjectButtonState::DownloadingBuildQueued : ProjectButtonState::Downloading);
         }
     }
 
-    void ProjectsScreen::HandleDownloadResult(const QString& /*projectName*/, bool /*succeeded*/)
+    void ProjectsScreen::HandleDownloadResult(const QString& projectName, bool succeeded)
```

Comment:
Similar comment to my previous one.

Reply:
Same as above, although in addition, a larger change for the DownloadController to use something like AZ::Event would fix this and simplify this in general for all object types.